src/components/AddToCart.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+async function AddToCart ({id}) { 
+try {
+    const response = await fetch("http://localhost:3000/order",{
```

Comment:
this needs to be  `fetch("/api/order"` since we are running the front end and back end on the same server